### ISSUE TITLE
Model Is Overfitting Data

### ISSUE THREAD

**Issue by dwij-patel123** [dwij-patel123/human_emotions_image_recognition#1]
as by looking at the predicted accuracy on the validation score and training data the accuracy differ by more than 13%.
